docs/evm_training/evmIntro.md

```diff
@@ -4,17 +4,50 @@ title: Nervos' Layer 2 EVM Training
 ---
 import useBaseUrl from "@docusaurus/useBaseUrl";
 
-Welcome to the Nervos' Layer 2 EVM (Ethereum Virtual Machine) Training! Here you will get an introduction to the Nervos ecosystem, learn about the various important concepts, and gain hands-on experience with the platform in preparation for deploying real-world applications.
+## Introduction to Nervos' Layer 2 EVM Training
+
+"Welcome to the Nervos' Layer 2 EVM (Ethereum Virtual Machine) Training! Here you will get an introduction to the Nervos ecosystem, learn about the various important concepts, and gain hands-on experience with the platform in preparation for deploying real-world applications."
+
+## About the Training Tasks and Compatibility
 
 To accomplish these goals, we have put together a set of tasks to guide you through the learning process. Each task will tell you what needs to be accomplished, and direct you to all the necessary tooling and informative materials you will need to do the job.
 
+<div class="task-wrapper">
+
+<h3> In this training, you'll learn how to: </h3>
+
+- [Create an account on the EVM Layer 2 Testnet](/evm_training/evmTask1.md)
+
+- [Deploy a Simple Ethereum Smart Contract on Polyjuice](/evm_training/evmTask2.md)
```

Comment:
```suggestion
- [Deploy a Simple Ethereum Smart Contract on Godwoken](/evm_training/evmTask2.md)
```

Polyjuice is Godwoken's evm backend.

Our current strategy is to mention it as little as possible, because although Godwoken is designed to support multiple VM Backends, it actually only supports EVM in the current stage.

Therefore, we weaken the concept of Polyjuice, and try to only mention Godwoken in various documents. Polyjuice will be not mentioned unless necessary.